R/count_labels.R

```diff
@@ -94,15 +96,17 @@ countBars <- function(inputId, choices, countsmax, countsnow = NULL) { # nolint
 #' Progress bar with label
 #'
 #' @param session (`session`) object passed to function given to `shinyServer`.
-#' @param inputId (`character(1)`) `shiny` id
-#' @param label (`character(1)`) Text to display followed by counts
-#' @param countmax (`numeric(1)`) maximal possible count for a single item.
+#' @param inputId (`character(1)`) `shiny` id.
+#' @param label (`character(1)`) Text to display followed by counts.
+#' @param countmax (`numeric(1)`) maximum possible count for a single item.
 #' @param countnow (`numeric(1)`) current count of a single item.
-#' @param counttotal (`numeric(1)`) total count to make whole progress bar
-#'  taking part of the container. Ratio between `countmax / counttotal`
-#'  determines `<style="width: <countmax / counttotal>%">`.
+#' @param counttotal (`numeric(1)`) total count to make whole progress bar taking part of the container. # TODO clarify
```

Comment:
Suggestion: Frame this as a sum of multiple `countmax` when there are more than a single item?